Description: Fix keepassxc missing libs
 .
 firejail (0.9.52-2foxxy3) testing; urgency=medium
 .
   * Fix keepassxc missing libraries.
Author: Lorenzo "Palinuro" Faletra <palinuro@foxxysec.org>
Last-Update: 2018-04-16

--- firejail-0.9.52.orig/etc/keepassxc.profile
+++ firejail-0.9.52/etc/keepassxc.profile
@@ -35,7 +35,7 @@ shell none
 
 private-bin keepassxc
 private-dev
-private-etc fonts,ld.so.cache,machine-id
+#private-etc fonts,ld.so.cache,machine-id
 private-tmp
 
 memory-deny-write-execute
